Sources/SwiftPhoenixClient/Socket.swift

```diff
@@ -780,6 +792,9 @@ public class Socket: PhoenixTransportDelegate {
   }
   
   public func onClose(code: Int) {
+    if code == CloseCode.normal.rawValue && self.closingAbnormally {
+      self.logItems("transport", "server confirmed client-initiated abnormal close")
+    }
```

Comment:
This is logged temporarily to allow client apps to track this edge case occurring in the wild.